osmnx/geometries.py

```diff
@@ -403,9 +404,9 @@ def _create_gdf(response_jsons, polygon, tags):
                     if multipolygon:
```

Comment:
@AtelierLibre Same comment here about the check as above regarding `linestring_or_polygon`

Reply:
@gboeing `pois` used to handle relations in a very different way, I suspect I just copied [line 344](https://github.com/gboeing/osmnx/blob/f05b359c418ff0139f6696d504b766a35775ef7c/osmnx/pois.py#L344) down when I was making the parsing of relations consistent with ways so the same comment as above applies.

Reply:
Resolved in 96a4489